server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -86,9 +88,19 @@ public long getTimeInMillis() {
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
-            out.writeVLong(current);
+            if (current < 0) {
+                logger.warn("'current' is negative [{}]", current);
```

Comment:
logging message is not very clear. Can we better qualify this message similar to:

```
SearchRequestStats 'current' went negative for phase 
```

Reply:
Also, do we really need this log message? If not, we can remove this

Reply:
Logs will have the class path for `PhaseStatsLongHolder` already

Reply:
Removed this log